win-spout-installer.nsi

```diff
@@ -61,10 +64,14 @@ Section "Spout 2 OBS Plugin" Section1
 	File "..\..\build64\plugins\win-spout\Release\win-spout.dll"
 	File "..\..\build64\plugins\win-spout\Release\win-spout.exp"
 	File "..\..\build64\plugins\win-spout\Release\win-spout.lib"
-	File "..\..\plugins\win-spout\deps\spout\SPOUT_LIBRARY\Binaries\x64\SpoutLibrary.dll"
-	File "..\..\plugins\win-spout\deps\spout\SPOUT_LIBRARY\Binaries\x64\SpoutLibrary.lib"
-	File "..\..\plugins\win-spout\deps\spout\VS2012\Binaries\x64\Spout.dll"
-	File "..\..\plugins\win-spout\deps\spout\VS2012\Binaries\x64\Spout.lib"
+	File "..\..\plugins\win-spout\deps\Spout2\BUILD\Binaries\x64\Spout.dll"
+	File "..\..\plugins\win-spout\deps\Spout2\BUILD\Binaries\x64\Spout.lib"
+	File "..\..\plugins\win-spout\deps\Spout2\BUILD\Binaries\x64\Spout_static.lib"
```

Comment:
Not sure we're using this are we? In fact, a while ago I think I realised that the `lib` files aren't even required in the installer, but lets fix that in a later PR